test/new-e2e/tests/security-agent-functional/security_agent_test.go

```diff
@@ -36,6 +36,7 @@ var (
 
 func TestVMSuite(t *testing.T) {
 	flake.Mark(t)
+	t.Parallel()
```

Comment:
I'm not sure I see the point, this suite has only one test anyway.

Side question: this test is marked as flaky currently, was this tested without the marker ?

Reply:
You're right I should not have kept it if it is the single test in the package. Removing